code/datums/paygrades/helper.dm

```diff
@@ -5,17 +5,20 @@
 
 	if(size)//Builds the prefix, if one should exist.
```

Comment:
This is technically where the if(P) thing should go, rather than doing another check later

Reply:
Leaving as is currently, messing with THAT is dangerous.